File: spring-r2dbc-querydsl-mysql-java/src/main/java/com/example/springr2dbcquerydslmysqljava/entity/Team.java
package com.example.springr2dbcquerydslmysqljava.entity;

import javax.persistence.Entity;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import org.springframework.data.annotation.Id;
import org.springframework.data.relational.core.mapping.Column;
import org.springframework.data.relational.core.mapping.Table;

@Table("team_tb")
@Entity
@NoArgsConstructor
@AllArgsConstructor
@Getter
public class Team {
    @Id
    @Column("id")
    private Long id;

    @Column("name")
    private String name;
}

File: spring-r2dbc-mysql-kotlin/src/main/kotlin/com/example/springr2dbcmysqlkotlin/entity/Team.kt
package com.example.springr2dbcmysqlkotlin.entity

import org.springframework.data.annotation.Id
import org.springframework.data.relational.core.mapping.Table

@Table("team")
data class Team(

    @Id    
    var id: Long,

    var teamName: String,
)
